CASC/PortfolioMode.cpp

```diff
@@ -577,7 +577,7 @@ void PortfolioMode::runSlice(std::string sliceCode, int timeLimitInDeciseconds)
   ASS_GE(sliceTime,0);
   try
   {
-    Options opt = *env.options;
+    Options& opt = *env.options;
```

Comment:
So this is now a reference to the post-fork `env.options`, right? But then why do we pass it around all the time? Couldn't it just be `env.options` everywhere?